package.json

```diff
@@ -49,5 +49,10 @@
     "pretty-quick": "^3.1.1",
     "rimraf": "^3.0.2",
     "serve": "^12.0.0"
+  },
+  "dependencies": {
+    "9000": "^0.1.3",
+    "kill-port": "^2.0.1",
+    "react-split": "^2.0.14"
```

Comment:
These are very unnecessary dependencies. Only install dependencies when given explicit permission to do so

Reply:
For reasons I don't know, unless I kill the port, running yarn dev always return an error message. 
See the attached screenshot. I will remove the "9000" and "kill-port" dependencies.

The "react-split" dependency on the other hand, was used in solving the task at hand.

![Screenshot 2022-11-14 101202](https://user-images.githubusercontent.com/33616969/202364373-6b579e5f-2074-41be-ab55-0d775e34bf92.jpg)